src/index.js

```diff
@@ -82,10 +84,19 @@ class HtmlWebpackEsmodulesPlugin {
         });
       }
       // Write it!
-      fs.writeFileSync(tempFilename, JSON.stringify(newBody));
+      const assetContents = JSON.stringify(newBody);
+      fs.writeFileSync(tempFilename, assetContents);
```

Comment:
It seems that `compilation.emitAsset` emits this tempFile for real. Do we still need to write it manually with `fs` at this point?

Reply:
I've removed this line in the latest commit